server/middlewares/auth.js

```diff
@@ -20,7 +19,7 @@ const isAuthorized = (req, res, next) => {
 const isAdmin = (req, res, next) => {
   const { email } = req.body;
   admin.findOne({ email }).then((data) => {
-    if (!data.email) {
```

Comment:
if the returned data doesn't contain an email the server crash and give me this error
```
(node:15545) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'email' of null
    at /home/mf/Desktop/Application system/applicants-system/server/middlewares/auth.js:22:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:15545) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:15545) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
````
so when there is no email found in admin data base the returned data will be null